jobs/reset.js

```diff
@@ -4,28 +4,107 @@ const dbs = require('../lib/dbs')
 const githubQueue = require('../lib/github-queue')
 const { createDocs } = require('../lib/repository-docs')
 
-module.exports = async function ({ repositoryFullName }) {
+module.exports = async function ({ repositoryFullName, accountId }) {
   repositoryFullName = repositoryFullName.toLowerCase()
-  // find the repository in the database
   const { repositories, installations } = await dbs()
-  const repoDoc = _.get(
-    await repositories.query('by_full_name', {
-      key: repositoryFullName,
-      include_docs: true
-    }),
-    'rows[0].doc'
-  )
+  const logs = dbs.getLogsDb()
+  const log = Log({ logsDb: logs, accountId: accountId, repoSlug: repositoryFullName, context: 'reset' })
+  log.info(`started reset`)
+
+  let repoDoc = ''
```

Comment:
Here and in L32, why have an empty string instead of undefined when both vars are objects?